src/stan/lang/function_signatures.h

```diff
@@ -785,8 +785,19 @@ for (size_t i=1; i < 10; i++) {
   add("num_elements", expr_type(int_type()), expr_type(vector_type(), i));
 }
 add("ordered_logistic_log", expr_type(double_type()), expr_type(int_type()), expr_type(double_type()), expr_type(vector_type()));
+add("ordered_logistic_log", expr_type(double_type()), expr_type(int_type(), 1), expr_type(vector_type()), expr_type(vector_type()));
```

Comment:
If possible, it'd be better to deal with the redundancy in a loop rather than a list.  Like loop over the int type and int container type, and vector and vector array type and then define everything overloaded inside the loop.  You can see how the other loops are defined in terms of what's in the container and what's being looped over.